tools/index.ts

```diff
@@ -9,21 +9,35 @@ import * as ts from "typescript";
 import KmpCompiler from "./kmp-compiler";
 import * as fs from "fs";
 import * as path from "path";
+import { createTrieDataStructure } from "./build-trie";
 
 export default class LexicalModelCompiler {
-  compile(o: LexicalModelSource) {
+  compile(modelSource: LexicalModelSource) {
     //
     // Load the model info file
     //
     let files = fs.readdirSync('../');
     let model_info_file = files.find((f) => !!f.match(/\.model_info$/));
 
-    //let model_info_file = fs.readdirSync('../').find((f) => !!f.match(/\.model_info$/));
     if(!model_info_file) {
       this.logError('Unable to find .model_info file in parent folder');
       return false;
     }
-    
+
+    /*
+     * Model info looks like this:
+     *
```

Comment:
Let's add a link to the documentation :) https://help.keyman.com/developer/cloud/model_info/1.0/